docs/mkdocs.yml

```diff
@@ -73,6 +73,7 @@ nav:
       - Task Lists: features/task_lists.md
       - Tags: features/tags.md
       - Error Handling: features/error_handling.md
+      - Middleware: features/middleware.md
```

Comment:
mmh :+1:

Reply:
can the doc be fully viewed locally btw ?

Reply:
Sure! See `docs/README.md`